### ISSUE TITLE
Error on Save Settings Email Notifications

### ISSUE THREAD

**Issue by superpikar** [solspace/craft-express-forms#5]
### Description
Hello, I got an error after save the value of Settings >> Email Notifications.

```
PHP Warning – yii\base\ErrorException
preg_match() expects parameter 2 to be string, array given
```
![Capture](https://user-images.githubusercontent.com/2832280/56421762-53c95800-62ce-11e9-9570-3ebe76d228eb.PNG)

I've install freeform before, so I want to use the template from the same directory. The email template directory is under `/templates/freeforms_email/`

### Steps to reproduce

1. Install Express Form
2. Go to Express Form >> Settings >> Email Notifications
3. Type folder name `freeforms_email/`
4. Hit "Save"

### Additional info

- Express Forms edition & version: Lite 1.0.0-beta.1
- Craft version: 3.1.22
- PHP version: 7.3
- Database driver & version:
- Other plugins & versions (if applicable): FreeForm Lite (trial) 2.5.17

**Comment by kjmartens** [comment]
Weird this seems to only affect some people. Anyway, the fix for this (available in next release) is:

Open **/vendor/solspace/craft-express-forms/src/services/EmailNotifications.php**, change this line:

```php
$files = $finder->name(['*.twig', '*.html'])
```

To this:

```php
$files = $finder
            ->name('*.twig')
            ->name('*.html')
```

Let me know if that doesn't resolve the issue for you. :)

**Comment by kjmartens** [comment]
This has been fixed now in 1.0.0-beta.2 :)
http://craft.express/forms/v1/setup/changelog.html#_1-0-0-beta-2-2018-04-19

**Comment by superpikar** [comment]
Thank you for the update :)
